app/models/bgs_dependents/adult_child_attending_school.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class AdultChildAttendingSchool < Base
+    attribute :first, String
+    attribute :middle, String
+    attribute :last, String
+    attribute :suffix, String
+    attribute :ssn, String
+    attribute :birth_date, String
+    attribute :ever_married_ind, String
+
+    def initialize(dependents_application)
+      @dependents_application = dependents_application
+      @was_married = @dependents_application.dig('student_address_marriage_tuition', 'was_married')
+      @name_and_ssn = @dependents_application['student_name_and_ssn']
+      self.attributes = child_attributes
+    end
+
+    def format_info
+      attributes.with_indifferent_access
+    end
+
+    def address
+      @dependents_application['student_address_marriage_tuition']['address']
+    end
+
+    private
+
+    def child_attributes
+      {
+        ssn: @name_and_ssn.dig('ssn'),
+        birth_date: @name_and_ssn.dig('birth_date'),
+        ever_married_ind: @was_married == true ? 'Y' : 'N'
+      }.merge(@name_and_ssn.dig('full_name'))
```

Comment:
Any of the attributes found using `dig` can return `nil` - is this OK? If not, might be a good idea to add some validations.

Reply:
Also j/w about why you're using `dig` as opposed to just referencing the the hash directly? Is `@name_and_ssn['ssn']`
not guaranteed to exist?